### PR TITLE
fix(core): invalid svg alignments in safari

### PR DESCRIPTION
**Pull request by splincode**
## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Build or CI related changes
- [ ] Tests related changes
- [ ] Documentation content changes

## What is the current behavior?
image

## What is the new behavior?

image

### CODE REVIEW

**Comment by waterplea** [comment]
Why did it use to work before? 🤔

**Comment by splincode** [comment]
We have problems only in Safari


Before:

image
image

After:

image
image

**Comment by waterplea** [comment]
I mean the arrow icon in back component. It's just a regular tui-svg, a lot of people use Safari, why haven't this been reported before? Is it only in back component? If so then why? Just curious.

**Comment by splincode** [comment]
I don't know((

File: projects/core/components/svg/svg.style.less [hunk 1]
```diff
@@ -12,11 +12,16 @@
...
+    ::ng-deep svg:not(.t-svg) {
+        width: inherit;
+        height: inherit;
+    }
 }
 
 .t-src {
     display: flex;
-    width: 100%;
+    width: inherit;
```
Comment (waterplea) [inline projects/core/components/svg/svg.style.less:24]: Why only width?

**Review by waterplea — APPROVED** [review]

File: projects/core/components/svg/svg.style.less (hunk 1 above)
Comment (splincode) [reply projects/core/components/svg/svg.style.less:24]: then many screenshots will be failed

**Review by nsbarsukov — APPROVED** [review]

**Pull request merged**